book/tutorials/basic-git.md

```diff
@@ -0,0 +1,201 @@
+# Basic Git/GitHub Skills Using `jupyterlab-git`
+
+```{admonition} Learning Objectives
+- Set up Git in the JupyterHub using [First time `git` setup](../preliminary/git)
+- Learn how to use the `jupyterlab-git` GUI in JupyterLab
+- Practice 4 basic Git/GitHub skills: cloning, committing, push/pull
+```
+
+```{important}
+Before we go over this tutorial we expect that you have already gone through this preparation material:
+- [JupyterHub login setup](../preliminary/jupyterhub)
+```
+
+## What is Git and GitHub?
+
+**Git** A program to track your file changes and create a history of those changes. Creates a 'container' for a set of files called a repository.
+
+**GitHub** A website to host these repositories and allow you to sync local copies (on your computer) to the website. *Lots* of functionality built on top of this.
+
+![](../img/git-repo-structure.png){width=200px}
```

Comment:
If you want to limit the width, you can use an image "directive" https://jupyterbook.org/en/stable/content/figures.html#images 

```{image} ../img/git-repo-structure.png
:alt: repo structure
:width: 200px
```